### PR TITLE
[MINOR]EDA-1368 new validations for jumping opponent's pawn

### PR DESCRIPTION
**Pull request by srobledo-eb**
I added validations so that a pawn can only jump two spaces when it has an enemy in front

### CODE REVIEW

File: game/pawn.py [hunk 1]
```diff
@@ -27,14 +27,70 @@ def __init__(self, color: str) -> None:
...
+    #         or (abs(from_col - to_col) <= 1 and (from_row - to_row) == 0)
+        # )
+
     def can_move(
+        self,
+        f_row: int,
+        f_col: int,
+        t_row: int,
+        t_col: int,
+        spots: list
```
Comment (ibrizuela-eb) [inline game/pawn.py:48]: Why are you using all the board spots if you in fact need just the element in the spot destination?

File: tests/test_pawn.py [hunk 2]
```diff
@@ -16,6 +17,40 @@ class MovementsTest(unittest.TestCase):
...
+        mock_one_move,
+    ):
+        mock_pawn = Pawn('blue')
+        spots = [None, None, None]
+        result = mock_pawn.can_move(1, 2, 3, 4, spots)
+        self.assertTrue(result)
+        mock_jump_horiz.assert_called_once()
+        mock_jump_vert.assert_called_once()
+        mock_one_move.assert_called_once()
+
```
Comment (ibrizuela-eb) [inline tests/test_pawn.py:53]: These both tests look identical, you should probably think in a parameterized test and replace de `assertTrue` or `assertFalse` for an `assertEqual(result, expected)`

File: game/pawn.py (hunk 1 above)
Comment (srobledo-eb) [reply game/pawn.py:48]: The spots are needed for two o the methods that are to be called inside

File: tests/test_pawn.py (hunk 2 above)
Comment (srobledo-eb) [reply tests/test_pawn.py:53]: I merged both into one function with the use of parameterized

**Review by ibrizuela-eb — APPROVED** [review]

**Pull request merged**